server/portal/settings/settings.py

```diff
@@ -44,8 +44,8 @@
 # whether the session cookie should be secure (https:// only)
 SESSION_COOKIE_SECURE = True
 CSRF_COOKIE_SAMESITE = 'Strict'
-# for local testing 
-CSRF_TRUSTED_ORIGINS = ['https://cep.test'] 
+# for local testing
+CSRF_TRUSTED_ORIGINS = settings_custom._CSRF_TRUSTED_ORIGINS
```

Comment:
Unless this settings will be required for all portals, can we include a default value as well?

Reply:
Updated to have default value